aplt/runner.py

```diff
@@ -313,9 +317,12 @@ def run_scenario(args=None, run=True):
 
     Usage:
         aplt_scenario WEBSOCKET_URL SCENARIO_FUNCTION [SCENARIO_ARGS ...]
-                      [--statsd_host STATSD_HOST]
-                      [--statsd_port STATSD_PORT]
-                      [--statsd_namespace STATSD_NAMESPACE]
+                      [--statsd_host=STATSD_HOST]
+                      [--statsd_port=STATSD_PORT]
+                      [--statsd_namespace=STATSD_NAMESPACE]
+                      [--datadog_api_key=DD_API_KEY]
```

Comment:
Ah, I do want to note that you need a namespace regardless of which one you use.